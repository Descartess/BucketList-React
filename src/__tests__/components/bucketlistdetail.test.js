import React from 'react';
import renderer from 'react-test-renderer';
import BucketListDetail from '../../components/bucketlistdetail';

describe('banner snapshot ', () => {
  const props = {
    onShowEditBucketList: jest.fn(),
    onShowDeleteBucketList: jest.fn(),
    selectBucketList: jest.fn(),
    name: '',
    id: 2,
  };
  it('renders correctly', () => {
    const rendered = renderer.create(
      <BucketListDetail {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
