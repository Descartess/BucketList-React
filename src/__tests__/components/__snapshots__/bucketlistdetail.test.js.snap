// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner snapshot  renders correctly 1`] = `
<button
  className="list-group-item"
  onClick={[Function]}
  type="button"
>
  
  <div
    className="pull-right"
  >
    <span
      className="glyphicon glyphicon-pencil"
      onClick={[Function]}
    />
    <span
      className="glyphicon glyphicon-trash"
      onClick={[Function]}
    />
  </div>
</button>
`;
